tox.ini

```diff
@@ -7,14 +7,27 @@ skipsdist=True
 
 [testenv]
 deps =
-	pytest
-    pre-commit
+  pytest
+  pre-commit
   pyrsistent
 commands =
-	pytest -v {posargs:tests}
+  pytest -v {posargs:tests}
 
 [testenv:docs]
 deps = sphinx
 changedir = docs
 commands =
     sphinx-build -b html -d build/doctrees source build/html
+
+[testenv:integration]
```

Comment:
Can we just noop the integration target so that travis still passes until we do?

We can have a local enabled target for testing of course.